Builds/test-all.sh

```diff
@@ -0,0 +1,38 @@
+#/bin/bash
+
+# Invoke as "bash ./Builds/test-all.sh"
+# or first make it executable ("chmod a+rx ./Builds/test-all.sh)
+#   then invoke as "./Builds/test-all.sh"
+#
+# Build must succeed without shell aliases for this to work. 
+
+BUILD="debug release all" 
+success=""
+scons ${BUILD}  && \
+  for dir in ./build/*
+  do
+    if [ ! -x ${dir}/rippled ]
```

Comment:
That's a great point, but after checking that it was `/bin/sh` compatible, I converted it down to not require `/bin/bash`.